src/CalcViewModel/Common/KeyboardShortcutManager.cpp

```diff
@@ -83,11 +82,13 @@ namespace CalculatorApp
             // The button will go into the visual Pressed state with this call 
             VisualStateManager::GoToState(button, "Pressed", true);
 
-            // This timer will fire after c_lightUpTime and make the button
+            // This timer will fire after lightUpTime and make the button
             // go back to the normal state. 
             // This timer will only fire once after which it will be destroyed
             auto timer = ref new DispatcherTimer();
-            timer->Interval = c_lightUpTime;
+            TimeSpan lightUpTime{};
+            lightUpTime.Duration = 500000L; // Half second (in 100-ns units)
```

Comment:
This change is surprising -- what was the warning that necessitated removing the constant for the magic number?

Reply:
It was complaining about changing the long to a value-type TimeSpan.